src/style.css

```
@@ -0,0 +1,88 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+* {
+  box-sizing: border-box;
+}
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir.ttc);
+}
+@font-face {
+  font-family: Avenir-Book;
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.form {
+  width: 100%;
+  position: relative;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-color: #fff;
+  margin-top: 20px;
+}
+
+.big_field {
+  height: 68px;
+  padding-left: 62px;
+  font-family: Avenir;
+  font-size: 16px;
+}
+
+.field {
+  outline: none;
+  border: none;
+  width: 100%;
+  color: #3d4e61;
+  border-radius: 4px;
+}
+
+.field::placeholder {
+  color: #3d4e61;
+}
+
+.big_btn_search {
```

не сокращай именна классов
```suggestion
.big_button_search {
```